### PR TITLE
ical events draft

### PR DESCRIPTION
**Pull request by evlli**

### CODE REVIEW

File: public/api/entropia-cloud-events-json-api.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?php
+
+use Sabre\VObject;
+
+class EntrioiaCloudEventsJsosApi
```
Comment (soc) [inline public/api/entropia-cloud-events-json-api.php:5]: typo? `Json*`

**Comment by jens1o** [comment]
I iterated on the approach here and have a working PoC in #55

**Comment by pascalwittler** [comment]
I have nothing against the integration of composer packages or libraries in general, but currently (for over a year) there is not even a reliable workflow for uploading some static files to the web server.

As long as the upload after changes here on Github is achieved by asking people you know and need to reach via chat or personal conversation to copy files by hand, I am against requiring further steps and installed software.

**Comment by jens1o** [comment]
Ah nice to know, thanks Pascal.

Who is in charge of running the web server? Can we integrate some pretty basic CI like doing rsync via ssh?

**Comment by pascalwittler** [comment]
> Who is in charge of running the web server?

As far as I know, at least @transcaffeine, @promasu and @stirlingzero have access to the web server. They can probably provide the full list of people who feel responsible and know what they are doing.

> Can we integrate some pretty basic CI like doing rsync via ssh?

I am asking for any automated deployment of the master branch to the web server for about a year now. Feel free to find people who are allowed to do things and do those things.

**Comment by promasu** [comment]
I don't have ✌️

**Comment by stirlingzero** [comment]
Afair, there was the idea for an automated process, but it seemed to be quite some work.
If you need a manual sync, just send me a message and I'll try to sync the project : )

**Comment by pascalwittler** [comment]
There is a list entry labeled "web" in the internal wiki that lists six people as maintainers, but I know that at least two of those people are no longer in the team, but another person who is not listed has already uploaded files by hand.

The fact that not even the information about who is responsible for creating a working process is easily accessible and up to date may be an indicator of the amount of work that needs to be done to automate deployment.

**Comment by jdreichmann** [comment]
I am for a webhook based workflow, where, after a merge, a script is executed which calls atleast `git fetch`.

**Comment by pascalwittler** [comment]
My suggestion for a very basic automated process, which I told several people over time, was to put a script on the web server running `git fetch`, called by a cronjob every five minutes or by a webhook after changes on the master branch. The answer I got several times was something like what @stirlingzero has already posted here (people are already working on something but it's complicated, …) but I would be very happy to see this become a common and uncomplicated solution now.

**Comment by jdreichmann** [comment]
I'll take care of it

**Pull request closed without merging**